### PR TITLE
Updates Non-Human Heads and Pay Modifiers

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

Updates non-human heads to give players more freedom and adjusts pay modifiers to be more reflective of lore

HoP can be lizard, plasmaman, and IPC now too
HoS can be phytosian, preternis, and polysmorph now too
Restrictions have been lifted for RD, CE, and CMO

Captain can still only be played by a human

Paycheck Modifier Changes overall:
- Lizard has a 0.85x paycheck modifier (from 0.5x)
- Phytosian has a 0.6x paycheck modifier (from 0.7x)
- Plasmman remains unchanged (1x)
- Mothperson remains unchanged (0.8x)
- Ethereal remains unchanged (0.7x)
- Preternis has a 0.6x paycheck modifier (from 0.8x)
- Polysmorph has a 0.3x paycheck modifier (from 0.6x)
- IPC has a 0.5x paycheck modifier (from 0.6x)

Also command footwraps were added back to dufflebags, because spares are nice

Also also plasmamen sec/heads now have their tracking beacons set to max by default

Sprites for new plasmamen stuff done by cuackles

# Why is this good for the game?

Player freedom and a wider potential of variety in theoretical station rosters creates interesting situations especially when someone might not be the most optimal species every for a role. If we cared about character balance on heads, a lot of quirks would be blacklisted on them (they aren't)

At the end of the day, it's a game, not an esport. A significant number of people do not care about roleplay, and thus do not care about lore (except when it suits their dated perspectives and biases). 

# Spriting
CMO plasmaman with only helmet and uniform on
image
image
image
image

HoP plasmaman with only helmet and uniform on
image
image
image
image

CMO icon sprite
image

HoP icon sprite
image

Command helm icon sprite
image

# Wiki Documentation

Paycheck modifiers to be changed on wiki
Available command roles to be changed on wiki

# Changelog

:cl:  Skrem_7, Cuackles
rscadd: Adds HoP plasmaman uniform, adds CMO uniform, adds plasmaman command envirohelm
tweak: Lizard, Plasmaman, and IPC added to HoP whitelist
tweak: Phytosian, Preternis, and Polysmorph added to HoS whitelist
tweak: CE, RD, and CMO have had their species restrictions lifted (except jellypeople [we'll get to you soon])
tweak: Several species have had their payment coefficients changed (often reduced) to reflect lore
tweak: All head clothing head duffle bags (except HoS) have a spare set of command footwraps now (again)
tweak: Plasmamen sec/heads will start with tracking beacon on
imageadd: Adds sprites for the plasmaman stuff
/:cl:

### CODE REVIEW

**Comment by iamthesilly** [comment]
aw, no more RD IPC?

**Comment by stwordcitizen** [comment]
not a big fan of phytosian hos because they will get easily killed by a welder or anyone with a laser weapon because of burn modifier and it completely changes the round when the hos gets killed by one traitor and now the killer has armoury access

**Comment by Marmio64** [comment]
While I recognize this would be perhaps more reflective of a lore I don't follow up on, it doesn't mean I have to like the changes and thus the lore, which I don't. Extraordinarily vulnerable races shouldn't be extraordinarily vital roles like the HoP and HoS, and I also just prefer HoP being human only.

**Comment by Skrem7** [comment]
> not a big fan of phytosian hos because they will get easily killed by a welder or anyone with a laser weapon because of burn modifier and it completely changes the round when the hos gets killed by one traitor and now the killer has armoury access

where's the laser weapon from if they don't have armory access? cargo? they might as well have armory access already

also phytosian warden and secoff are fine i guess but hos is just too much

**Comment by Skrem7** [comment]
> While I recognize this would be perhaps more reflective of a lore I don't follow up on, it doesn't mean I have to like the changes and thus the lore, which I don't. Extraordinarily vulnerable races shouldn't be extraordinarily vital roles like the HoP and HoS, and I also just prefer HoP being human only.

i should be a head of staff with a crippling negative quirk because i can

**Comment by stwordcitizen** [comment]
> > not a big fan of phytosian hos because they will get easily killed by a welder or anyone with a laser weapon because of burn modifier and it completely changes the round when the hos gets killed by one traitor and now the killer has armoury access
> 
> where's the laser weapon from if they don't have armory access? cargo? they might as well have armory access already
> 
> also phytosian warden and secoff are fine i guess but hos is just too much

Just because when you kill the hos with a welder you get bridge access, emp gun, good armour, lots of access and sec gear.

**Comment by Skrem7** [comment]
> Just because when you kill the hos with a welder you get bridge access, emp gun, good armour, lots of access and sec gear.

if you die to a welder as a phytosian hos then you are unrobust, you have melee armor, wound armor, a shield, an energy gun, the hos gun, a baton, and all the rest of the sec gear known to mankind

quite literally a player issue

**Comment by iamthesilly** [comment]
"don't let people have races with disadvantages as sec" mfs when not all of sec is the same race and their greatest tool is teamwork

**Comment by stwordcitizen** [comment]
> > Just because when you kill the hos with a welder you get bridge access, emp gun, good armour, lots of access and sec gear.
> 
> if you die to a welder as a phytosian hos then you are unrobust, you have melee armor, wound armor, a shield, an energy gun, the hos gun, a baton, and all the rest of the sec gear known to mankind
> 
> quite literally a player issue

Shouldn't make it easier to kill the most powerful role on the station with the most powerful gear because of a racial damage modifier

**Comment by Skrem7** [comment]
> Shouldn't make it easier to kill the most powerful role on the station with the most powerful gear because of a racial damage modifier

It is someone's choice to have a disadvantage as a species.

If we want to talk "don't let ppl be easy to kill" maybe we should put a whitelist on head roles, and you have to prove robustness in order to play it. Otherwise, we could have the terrible situation where a HoS is too easy to kill.

**Comment by stwordcitizen** [comment]
> > Shouldn't make it easier to kill the most powerful role on the station with the most powerful gear because of a racial damage modifier
> 
> It is someone's choice to have a disadvantage as a species.
> 
> If we want to talk "don't let ppl be easy to kill" maybe we should put a whitelist on head roles, and you have to prove robustness in order to play it. Otherwise, we could have the terrible situation where a HoS is too easy to kill.

I think its borderline griefing to disadvantage yourself with 2x or 3x modifiers to damage as any command role. Don't think that whitelist is neccessary I just don't like the idea of hos having a huge disadvantage in 2 or 3x burn modifier

**Comment by 00ze-cyclone** [comment]
> > > Shouldn't make it easier to kill the most powerful role on the station with the most powerful gear because of a racial damage modifier
> > 
> > 
> > It is someone's choice to have a disadvantage as a species.
> > If we want to talk "don't let ppl be easy to kill" maybe we should put a whitelist on head roles, and you have to prove robustness in order to play it. Otherwise, we could have the terrible situation where a HoS is too easy to kill.
> 
> I think its borderline griefing to disadvantage yourself with 2x or 3x modifiers to damage as any command role. Don't think that whitelist is neccessary I just don't like the idea of hos having a huge disadvantage in 2 or 3x burn modifier

Worst burn mod is phytosians with x2 and they get a self regen so if they use their gear to not immediately die they just go back to full health for free, beside who care if sec isn’t perfectly ultra gamer mode to steamroll antags.

**Comment by iamthesilly** [comment]
Keep in mind that security's greatest strength is in cooperation and looking after each other, so one person having a weakness to something doesn't mean all of security is doomed.

**Comment by MajManatee** [comment]
In favor because hugboxing is dumb.

Letting races with inherent downsides be played if players want to, should be allowed especially if it's within the lore. 
If those people want to give themselves a challenge then go for it. They know the risks.
If a plant hos dies to some assistant with a welder then so be it, they signed up and knew the risks.

**Comment by CrazyDrakkon** [comment]
Can't wait to not afford a bag of chips again!!!

**Comment by Moltijoe** [comment]
God bless, more reason to be violently racist towards humans as a preternis.
Also, whats the reasoning behind no IPC RD?

**Comment by Skrem7** [comment]
> Also, whats the reasoning behind no IPC RD?

Robot in charge of robots = yikes?

Mostly based in the SELF incident but tbh if IPC can be all that other stuff it doesn't make the most sense; I can roll back the reversion if that'd be popular, since Sapphic and others have brought it up.

**Comment by Moltijoe** [comment]
> Robot in charge of robots = yikes?
> 
> Mostly based in the SELF incident but tbh if IPC can be all that other stuff it doesn't make the most sense; I can roll back the reversion if that'd be popular, since Sapphic and others have brought it up.

Once the MALF rework gets in and SELF is actually added i suppose i can understand the RD thing, but for now, has the SELF incident actually happened yet on the station? I assumed it was just a looming possible threat.

**Comment by Skrem7** [comment]
> > Robot in charge of robots = yikes?
> 
> > 
> 
> > Mostly based in the SELF incident but tbh if IPC can be all that other stuff it doesn't make the most sense; I can roll back the reversion if that'd be popular, since Sapphic and others have brought it up.
> 
> 
> 
> Once the MALF rework gets in and SELF is actually added i suppose i can understand the RD thing, but for now, has the SELF incident actually happened yet on the station? I assumed it was just a looming possible threat.
> 
> 
> 
> 

There's a bus antag that's a SELF agent that can (should) only be IPC. Happens on MALF, but not naturally.

**Comment by Moja10** [comment]
This needs to be atomized to shit, lizard CE is fine but I would really appreciate if we didn't have plant HoS running around dying to 3 laser gun shots.

**Comment by neeshacark** [comment]
plant hos is fine, people like to go 'but muh laser 3 shot' but if someone has a laser killing the hos wont give them any benefit because if they have a laser they almost certainly have all/most access at this point because lasers come from the armory.  syndicate weapons are mostly brute based, which pods dont give two shits about because they heal brute ezpz.  infact, most antag weapons are brute or at minimum NOT burn.

if thats somehow still an issue because 'sec shouldnt die so easy' then you should remove plant secoff/warden. and if its 'heads shouldnt die easy' then they shouldnt be able to be heads.

on the flip side, plantpeople get passive healing which is a VERY USEFUL THING for a hos to have and makes a lot of sense. + you also act like the entirety of security hinges on one person (it doesn't) and you could make the 'but the armory access' but again - if in this argument it's only an issue bcuz "hos ez to kill with laser" then your armory is already breached.  if it's a issue of welders i'd like to ask you to watch antags with welders run up to venus and then get stamcritted before they can do anything noticeable. 

lastly, this also ignores the part where we have like 5 podperson mains and only a handful of that handful even play sec. I think that's mostly Venus and I who I ever see on sec.  You probably won't see it much but for those who do it benefits a different playstyle and is nowhere near as much as a downside as people like to make it out to be.

tl;dr - plantperson hos is fine, if you have lasers as antag you almost certainly have AA or can get it easily. plant hos has much more benefits and the downsides are negligible

**Comment by Mqiib** [comment]
basically what cark said

**Comment by Therandomhoboo** [comment]
Ya'll need to stop getting too deep about restricting roles to certain races. Let people do it and if they die its a literal player skill issue. It aint that deep guys, chill its a fucking space game

**Review by Moltijoe — APPROVED** [review]
all of these are pretty shrimple changes

**Review by cuackles — APPROVED** [review]
Phone review phone review phone review, 4 new sprites icons do be looking kinda iffy but they work as a stand in and i will revisit them once I'm home ( if i don't forget )( i will set a reminder with the yogs bot don't worry ) ( third parenthesis for the charm )

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@ DISREGUARD THIS FILE IF YOU'RE INTENDING TO CHANGE ASPECTS OF PLAYER CONTROLLED
 	siemen_coeff = 0.75 //I wouldn't make semiconductors out of plant material
```
Comment (MajManatee) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm:23]: scroll up a bit to line 3 of this file and let me know if thats right or not

Comment (Skrem7) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm:23]: no clue, but changes here have been popping up in game made on this file

**Comment by ToasterBiome** [comment]
non-human HOP moment

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm (hunk 1 above)
Comment (Moltijoe) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm:23]: because there's literally duplicate creations of /datum/species/pod

**Comment by N3D6** [comment]
> non-human HOP moment

idc about the other stuff but this is ass

**Comment by MajManatee** [comment]
The time of man has come to an end.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**